test/bind.sh

```diff
@@ -1,23 +1,39 @@
 #!/bin/bash -e
 
-. shared.sh
+INSTANCE_ID=$1
+BINDING_ID=$2
+PLAN_UUID="7f4a5e35e4af2beb70076e72fab0b7ff"
```

Comment:
This looks like it is hardcoded I assume there is a registry config that we must use to get this to work. I think it might be worth documenting in the README.

Reply:
@shawn-hurley heh it was hardcoded because on a few of the calls we don't really care apparently. Clearly this is NOT going to work with provision. I'll look into it.